slides/intro-02-data-budget.qmd

```diff
@@ -224,32 +190,25 @@ There are `r nrow(taxi_test)` rows and `r ncol(taxi_test)` columns in the test s
 
 *Try out different values in `set.seed()` to see how the results change.*
 
+<br>
+
+*Extension/Challenge: This is a simple random split. Which other types of splits can you think of and does rsample offer corresponding functions?*
+
 ```{r ex-try-splitting}
 #| echo: false
 countdown::countdown(minutes = 5, id = "try-splitting")
 ```
 
 ## Data splitting and spending `r hexes("rsample")`
 
-```{r taxi-split-prop}
+```{r cat-split-prop}
 set.seed(123)
-taxi_split <- initial_split(taxi, prop = 0.8)
-taxi_train <- training(taxi_split)
-taxi_test <- testing(taxi_split)
+cat_split <- initial_split(cat_adoption, prop = 0.8)
```

Comment:
Should we mention that, if we don't make the Surv object at first, we could use the event indicator for stratification then make the surv object later? 

We could put something in the annotations with a recipe that would remove/change roles of the original columns used to make the Surv object?

Reply:
I've turned this into #10 because I want to see if we can come up with a clear recommendation for which possible path to take: do we still make the Surv object first (probably?), do we remove the time and event via `step_rm()` or "just" update the role (and possibly the role requirements)? How do we weave recipes in there at this stage? Do we want to point to the rsample issue for feedback?